### ISSUE TITLE
Add "Minimum Number of Chapters" as a Search Filter

### ISSUE THREAD

**Issue by oinoirenigonasoindoisanfoidsn** [KotatsuApp/Kotatsu#1183]
### Describe your suggested feature

It would be useful to have a filter for the minimum number of chapters in the manga search functionality. This feature would allow users to find manga with enough content to suit their preferences, such as ensuring the series has a substantial number of chapters before starting.

Use case: Users searching for new manga to read often prefer series that are more developed or provide a longer reading experience. Adding this filter would improve discoverability for such users by allowing them to specify a threshold for the number of chapters during their search.

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.
